cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -347,7 +347,7 @@ public CohortEngineRestHandler() {
 			@ApiImplicitParam(name=CQL_DEFINITION, value = CQL_REQUIREMENTS,dataTypeClass = File.class, required=true, paramType="form", type="file" )
 	})
 	@ApiResponses(value = {
-			@ApiResponse(code = 200, message = "Successful Operation"),
+			@ApiResponse(code = 200, message = "Successful Operation: The response object is a json payload with the list of passing patients: {\"result\": [\"patient1\", \"patient2\"] }"),
```

Comment:
My previous comment was to imply that we should create a simple response object and use it here.